app/services/object_file_validator.rb

```diff
@@ -27,15 +27,11 @@ def validate
     errors[:empty_files] = empty_files if empty_files > 0
     errors.merge!(registration_check) # needs to come before manifest checks
     if using_file_manifest? # if we are using a file manifest, let's add how many files were found
-      batch_id = object.druid.id
-      if batch_id && file_manifest.manifest[batch_id]
-        manifest_files = file_manifest.manifest[batch_id].fetch(:file_sets, []).flat_map { |_seq, val| val[:files] }
-        counts[:files_in_manifest] = manifest_files.count
-        relative_paths = object.object_files.map(&:relative_path) # Files found on disk
-        all_paths = (cocina_filenames + relative_paths).uniq
-        counts[:files_found] = (manifest_files.pluck(:filename) & relative_paths).count
+      if object.druid.id && file_manifest.manifest[object.druid.id]
+        counts[:files_in_manifest] = manifest_filenames.count
+        counts[:files_found] = (manifest_filenames & on_disk_filenames).count
```

Comment:
Is the `&` supposed to be a `+` here?

Reply:
While I admit that "files found" is completely ambiguous, the counting logic was not actually changed in this PR, just the names of the variables. Files found is the intersection of the files in the manifest and files on disk.